internal/docs/exported_fields.go

```diff
@@ -70,7 +70,10 @@ func renderFieldsTable(builder *strings.Builder, collected []fieldsTableRecord)
 	builder.WriteString("\n")
 	for _, c := range collected {
 		description := strings.TrimSpace(strings.ReplaceAll(c.description, "\n", " "))
-		builder.WriteString(fmt.Sprintf("| %s | %s | %s |", c.name, description, c.aType))
+		builder.WriteString(fmt.Sprintf("| %s | %s | %s |",
+			strings.ReplaceAll(c.name, "*", "\\*"),
+			strings.ReplaceAll(description, "*", "\\*"),
```

Comment:
Nit. Consider using a [Replacer](https://pkg.go.dev/strings#NewReplacer), so we only need to keep a list of replacements.

Reply:
Fixed